airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -126,3 +148,28 @@ def spec(self, *args, **kwargs) -> ConnectorSpecification:
                 ),
             ),
         )
+
+    def _update_insights_streams(self, insights, args, streams) -> List[Type[Stream]]:
```

Comment:
I suggest the following approach for these custom streams: 

1. The "standard" streams offered by the connector are always available (if a user doesn't want them they can always just deselect them)
2. If the user inputs any custom streams, they are named as `custom_<user-input-name>` and appended to the list of streams

This way it is very very obvious to the user what is happening. This is especially important as the connector's config is updated over time e.g: a user might call a stream `ads_insights` today, then remove it next week, and now the data represented in that stream is mixed with the "standard" ads insight stream and the "custom" insight stream. Making the suggestion above to make such situations much less likely.